tests/PHPUnit/schemas/ThingTest.php

```diff
@@ -218,6 +243,20 @@ public function testBuildReturnsFromCache() {
 		$method->setAccessible( true );
 
 		M::userFunction( 'wp_cache_get', array(
+			'times'  => 1,
+			'args'   => array( 'schema_123_last_update', 'schemify', false ),
+			'return' => 200,
+		) );
+
+		M::userFunction( 'wp_cache_get', array(
+			'times'  => 1,
```

Comment:
To be honest, I'm not a huge  fan of WP_Mock, it's boring to mock all those basic functions and I always have the feeling that the tests are biased and it's not really testing anything, The only reason I added `times` was because I was seeing it being used on across tests. I'm totally fine removing it. if you think it makes sense :)

For these tests, if the function call doesn't happen the test will fail anyway, so `times` is not that important here.